packages/perseus-core/src/validation.types.ts

```diff
@@ -378,25 +379,6 @@ export interface ValidationDataTypes {
     plotter: PerseusPlotterValidationData;
 }
 
-/**
- * A map of validation data, keyed by `widgetId`. This data is used to check if
- * a question is answerable. This data represents the minimal intersection of
- * data that's available in the client (widget options) and server (scoring
- * data) and is represented by a group of types known as "validation data".
- *
- * NOTE: The value in this map is intentionally a subset of WidgetOptions<T>.
- * By using the same shape (minus any unneeded data), we are able to pass a
- * `PerseusWidgetsMap` or ` into any function that accepts a
- * `ValidationDataMap` without any mutation of data.
- */
-export type ValidationDataMap = {
```

Comment:
I just didn't see the point of having this type and it was keeping me from reusing helpers between scoring and validation logic. Ideally in the future we could just have a AnswerfulPerseusItem and AnswerlessPerseusItem or something.